corehq/apps/analytics/README.md

```diff
@@ -103,3 +103,8 @@ Generally available in areas of interest to the product and growth teams: signup
 ### Facebook Pixel
 
 Their script is included in [signup](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/registration/templates/registration/register_new_user.html), but we don't do any event tracking or other interaction with it. Very little related code, just the script inclusion.
+
+### Google Tag Manager (GTM)
+
+This script is available in the [gtm.js](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/analytics/static/analytix/js/gtm.js).Related code is just for sending the user properties to GTM.
```

Comment:
nit: Its script is available?

Reply:
nit: what is referred to from "related code"?

Reply:
Addressed in https://github.com/dimagi/commcare-hq/pull/34554/commits/a15d0df1e5a5dafe0abac2fbbc058b9ed80559f2.
"related code" is just the script code.